### ISSUE TITLE
Űrbázis 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#24]
```
Game version:0.92

Nem lehet űrbázissal felszerelni bolygót, ha nincs katonai űrkikötő az 
adott bolygón, míg az eredeti játékban lehet.
```

Original issue reported on code.google.com by `Jozsef.T...@gmail.com` on 16 Aug 2011 at 9:50

**Comment by GoogleCodeExporter** [comment]
```
Ez vonatkozik a vadászokra és a járművekre is?
```

Original comment by `akarn...@gmail.com` on 16 Aug 2011 at 10:58

**Comment by GoogleCodeExporter** [comment]
```
Csak vadászokat nem lehet telepíteni űrkikötő nélkül, illetve az 
űrkikötők száma szabályozza a flották számát is.
```

Original comment by `Jozsef.T...@gmail.com` on 16 Aug 2011 at 5:43

**Comment by GoogleCodeExporter** [comment]
```
Átalakítom a képernyőt, hogy a vadászokhoz kelljen csak űrkikötő. Az 
űrállomásokhoz és járművekhez nem kell. A flottakorlátban nem vagyok 
biztos, jelenleg űrkikötő kell egy új flotta létrehozásához, de más 
nem. Szerintem nincs nagy jelentősége, mert anno legfeljeb 1-3 flottával 
nyomultam.
```

Original comment by `akarn...@gmail.com` on 16 Aug 2011 at 7:32

**Comment by GoogleCodeExporter** [comment]
```
Fix 0.93.100
```

Original comment by `akarn...@gmail.com` on 17 Aug 2011 at 11:08
- Changed state: **Fixed**
